services/API-service/src/api/metadata/metadata.service.ts

```diff
@@ -16,6 +17,21 @@ export class MetadataService {
   @InjectRepository(LayerMetadataEntity)
   private readonly layerRepository: Repository<LayerMetadataEntity>;
 
+  public constructor(
+    private readonly disasterTypeService: DisasterTypeService,
+  ) {}
+
+  public async getMainExposureIndicatorMetadata(
+    disasterType: DisasterType,
+  ): Promise<IndicatorMetadataEntity> {
+    return await this.indicatorRepository.findOne({
+      where: {
+        name: (await this.disasterTypeService.getDisasterType(disasterType))
```

Comment:
Nice to have: Use `getMainExposureIndicator` instead.

Reply:
done